Makefile

```
@@ -24,7 +24,7 @@ website/local:
 .PHONY: website/build-local
 website/build-local:
 	@echo "==> Building local Docker image"
-	@docker build https://github.com/hashicorp/terraform-website.git\#master \
+	@docker build https://github.com/hashicorp/terraform-website.git\#kevin/terraform-split \
```

Apply this change when https://github.com/hashicorp/terraform-website/pull/2228 is merged
```suggestion
	@docker build https://github.com/hashicorp/terraform-website.git\#master \
```